include/eve/function/shuffle.hpp

```diff
@@ -7,28 +7,30 @@
 #pragma once
 
 #include <eve/detail/overload.hpp>
-#include <eve/detail/function/patterns.hpp>
-#include <eve/detail/function/swizzle.hpp>
+#include <eve/module/real/core/detail/swizzle.hpp>
 
 namespace eve
 {
   EVE_MAKE_CALLABLE(shuffle_, shuffle);
+}
+
+
+#include <eve/module/real/core/detail/generic/patterns.hpp>
```

Comment:
Why are patterns in detail?

Reply:
Ah.

Seems like 'patterns' maybe should be renamed as 'find_optimized_pattern`? Or `find_optimized_shuffle_pattern`?

Reply:
Yes I was thinking about that to prevent confusion